stages/02_process_embl.py

```diff
@@ -0,0 +1,11 @@
+from Bio import SeqIO
+import pandas as pd
+
+new_embl_records = SeqIO.parse("./download/miRNA.dat", "embl")
+# prev_embl_records = SeqIO.parse("./download/miRNA.dead", "embl")
+records = [{'name': record.name, 'description': record.description} for record in new_embl_records]
+# old_records = [{'name': record.name, 'description': record.description} for record in prev_embl_records]
+df = pd.DataFrame.from_records(records)
+# dead = pd.DataFrame.from_records(old_records)
+df.to_parquet("./brick/miRNA.parquet")
```

Comment:
Let's call this `miRNA.dat.parquet`. It helps with tracking back to which file it was created from. The others would be `miRNA.dead.parquet`, but we won't process that yet.